test/Renci.SshNet.IntegrationTests/Dockerfile.TestServer

```diff
@@ -10,7 +10,8 @@ RUN apk update && apk upgrade --no-cache && \
     apk add --no-cache openssh && \
     # install openssh-server-pam to allow for keyboard-interactive authentication
     apk add --no-cache openssh-server-pam && \
-    dos2unix /etc/ssh/* && \
+    # must not use * for dos2unix parameter otherwise it tries to process folders too and fails
+    dos2unix /etc/ssh/ssh*key && \
```

Comment:
Maybe there is a new alpine image and it has different behavior, but I don't have a problem with this change as long as it works.